src/os/ui/menu/menuitem.js

```diff
@@ -187,15 +187,19 @@ class MenuItem {
       return html;
     }
 
+    var classes = [];
+    var isItem = type !== types.SUBMENU && type !== types.GROUP && type !== types.SEPARATOR;
+
     html += '<li';
     if (type === types.SEPARATOR) {
-      html += '>-</li>';
-      return html;
+      if (this.label) {
+        classes.push('dropdown-header');
```

Comment:
The `SEPARATOR` type is for rendering a line between items, and this seems to be duplicative of the `GROUP` type. Does this add anything over using `GROUP`?

Reply:
Switched these all over to use `GROUP` and added a breaking change commit.